geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/iterators/Z3IteratorTest.scala

```diff
@@ -55,19 +50,16 @@ class Z3IteratorTest extends Specification {
       override def hasTop: Boolean = staged != null
     }
 
-    val sfc = LegacyZ3SFC(TimePeriod.Week)
-
-    "iterate on points" >> {
-      val (xmin, ymin, tmin) = sfc.index(lx, ly, lt).decode
-      val (xmax, ymax, tmax) = sfc.index(ux, uy, ut).decode
-
-      val iter = new Z3Iterator
+    val sft = SimpleFeatureTypes.createType("z3IteratorTest", "dtg:Date,*geom:Point:srid=4326")
+    val filter = ECQL.toFilter("bbox(geom, -78, 38, -75, 40) AND dtg DURING 1970-01-01T00:05:00.000Z/1970-01-01T00:15:00.000Z")
```

Comment:
Would it make sense to add a case or three to this test which has a duration longer than the 2x the Z3.period?

Reply:
i'm not crazy about this test because you have to construct everything by hand, which means you have to set everything up correctly. seems like it's tested better in the full query tests, which do already cover that scenario